packages/create-cloudflare/src/workers.ts

```diff
@@ -1,194 +1,65 @@
-	await copyExistingWorkerFiles(ctx);
-	await updateFiles(ctx);
-	await offerGit(ctx);
-	endSection("Application created");
-
-	startSection("Installing dependencies", "Step 2 of 3");
-	chdir(ctx.project.path);
-	await npmInstall();
-	await installWorkersTypes(ctx);
-	await gitCommit(ctx);
-	endSection("Dependencies Installed");
-	if (!preexisting) {
-		await offerToDeploy(ctx);
-		await runDeploy(ctx);
-	}
-
-	await printSummary(ctx);
+export const readWranglerToml = async (ctx: C3Context) => {
+	const wranglerTomlPath = resolve(ctx.project.path, "wrangler.toml");
+	return readFile(wranglerTomlPath);
 };
 
-async function getTemplate(ctx: Context) {
-	const preexisting = ctx.args.type === "pre-existing";
-	const template = preexisting ? "hello-world" : ctx.args.type;
-	const path = resolve(
-		// eslint-disable-next-line no-restricted-globals
-		__dirname,
-		"..",
-		"templates",
-		template,
-		ctx.args.ts ? "ts" : "js"
-	);
-
-	return { preexisting, template, path };
-}
-
-async function copyFiles(ctx: Context) {
-	const { template, path: srcdir } = await getTemplate(ctx);
-	const destdir = ctx.project.path;
-
-	// copy template files
-	updateStatus(`Copying files from "${template}" template`);
-	await cp(srcdir, destdir, { recursive: true });
-
-	// reverse renaming from build step
-	await rename(join(destdir, "__dot__gitignore"), join(destdir, ".gitignore"));
-}
-
-async function copyExistingWorkerFiles(ctx: Context) {
-	const { preexisting } = await getTemplate(ctx);
-
-	if (preexisting) {
-		await chooseAccount(ctx);
-
-		if (ctx.args.existingScript === undefined) {
-			ctx.args.existingScript = await processArgument<string>(
-				ctx.args,
-				"existingScript",
-				{
-					type: "text",
-					question:
-						"Please specify the name of the existing worker in this account?",
-					label: "worker",
-					defaultValue: ctx.project.name,
-				}
-			);
-		}
+export const writeWranglerToml = async (ctx: C3Context, contents: string) => {
+	const wranglerTomlPath = resolve(ctx.project.path, "wrangler.toml");
+	return writeFile(wranglerTomlPath, contents);
+};
 
-		// `wrangler init --from-dash` bails if you opt-out of creating a package.json
-		// so run it (with -y) in a tempdir and copy the src files after
-		const tempdir = await mkdtemp(
-			join(tmpdir(), "c3-wrangler-init--from-dash-")
-		);
-		await runCommand(
-			[
-				...dlx,
-				"wrangler@3",
-				"init",
-				"--from-dash",
-				ctx.args.existingScript,
-				"-y",
-				"--no-delegate-c3",
-			],
-			{
-				silent: true,
-				cwd: tempdir, // use a tempdir because we don't want all the files
-				env: { CLOUDFLARE_ACCOUNT_ID: ctx.account?.id },
-				startText: "Downloading existing worker files",
-				doneText: `${brandColor("downloaded")} ${dim(
-					`existing "${ctx.args.existingScript}" worker files`
-				)}`,
-			}
-		);
+/**
+ * Update the `wrangler.toml` file for this project by setting the name
+ * to the selected project name and adding the latest compatibility date.
```

Comment:
```suggestion
 * to the selected project name and adding or replacing the compatibility date to the latest.
```

Reply:
This code no longer replaces an explicit date.